### PR TITLE
Created tab and fields for Sales Force.

### PR DESCRIPTION
**Pull request by waheedahmed**
[ECOM-5110](https://openedx.atlassian.net/browse/ECOM-5110)

**Acceptance Criteria:**
1. VERIFY that data for all fields required for publishing to Salesfore are shown as a tab on the course-run details page.
2. VERIFY that copy to clipboard can be done using a copy button for each field (rather than highlighting manually and copying)

**Screenshots:**
screencapture-localhost-8004-publisher-course_runs-17-1473253006332

### CODE REVIEW

**Comment by waheedahmed** [comment]
@awais786 @bderusha Please review.

**Comment by awais786** [comment]
@bderusha Please review.

File: course_discovery/apps/publisher/tests/test_wrapper.py [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # pylint: disable=no-member
 import ddt
+from datetime import datetime, timedelta
 from unittest import mock
-
 from django.test import TestCase
 
+from course_discovery.apps.course_metadata.models import CourseRun as CourseMetadataCourseRun
```
Comment (bderusha) [inline course_discovery/apps/publisher/tests/test_wrapper.py]: Can we import the Pacing directly here? This import is very broad

**Comment by bderusha** [comment]
:+1: once Todos are resolved

File: course_discovery/apps/publisher/tests/test_wrapper.py (hunk 1 above)
Comment (awais786) [reply course_discovery/apps/publisher/tests/test_wrapper.py]: @bderusha I think we need to import the CourseRun here. Pacing directly not working.

**Comment by awais786** [comment]
@bderusha Would you please tell me what the Todos ? I did't find any information on the ticket.

File: course_discovery/templates/publisher/course_run_detail.html [hunk 2]
```diff
@@ -12,6 +12,7 @@
         <button data-tab="#tab-2">{% trans "STUDIO" %}</button>
         <button data-tab="#tab-3">{% trans "CAT" %}</button>
         <button data-tab="#tab-4">{% trans "DRUPAL" %}</button>
+        <button data-tab="#tab-5">{% trans "Sales Force" %}</button>
```
Comment (bderusha) [inline course_discovery/templates/publisher/course_run_detail.html]: `Salesforce`  One word

File: course_discovery/templates/publisher/course_run_detail.html [hunk 3]
```diff
@@ -61,6 +62,9 @@ <h2 class="hd-2 emphasized">
         <div id="tab-4" class="tab-content">
             {% include 'publisher/course_run_detail/_drupal.html' %}
         </div>
+        <div id="tab-5" class="tab-content">
+            {% include 'publisher/course_run_detail/_sales_force.html' %}
```
Comment (bderusha) [inline course_discovery/templates/publisher/course_run_detail.html]: same here `_salesforce.html` one word

File: course_discovery/templates/publisher/course_run_detail/_sales_force.html [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+            <div class="heading">
+                {% trans "Account" %}
+                {% include "publisher/course_run_detail/clipboard.html" %}
+            </div>
+            <div class="copy">{{ object.organization_name }}</div>
+        </div>
+        <div class="info-item">
+            <div class="heading">
+                {% trans "Authored in Studio?" %}
+                {% include "publisher/course_run_detail/clipboard.html" %}
```
Comment (bderusha) [inline course_discovery/templates/publisher/course_run_detail/_sales_force.html]: Clipboard not required for booleans.  Please update here and everywhere a boolean is presented

File: course_discovery/apps/publisher/wrappers.py [hunk 5]
```diff
@@ -131,3 +140,39 @@ def workflow_state(self):
...
+    def is_multiple_partner_course(self):
+        organizations_count = self.wrapped_obj.course.organizations.all().count()
+        if organizations_count > 1:
+            return True
+
+        return False
+
+    @property
+    def is_self_paced(self):
+        if self.wrapped_obj.pacing_type == 'self_paced':
```
Comment (bderusha) [inline course_discovery/apps/publisher/wrappers.py]: please use `CourseMetadataCourseRun.Pacing.Self`

**Review by bderusha — CHANGES_REQUESTED** [review]
Looks good.  Will review again when all TODOs are addressed in the code

**Comment by waheedahmed** [comment]
@bderusha All TODO's resolved and addressed your comments please take another look.

**Comment by awais786** [comment]
👍

**Comment by bderusha** [comment]
👍

**Review by bderusha — APPROVED** [review]

**Pull request merged**